README.md

```diff
@@ -0,0 +1,53 @@
+# Welcome to cascade 👋
+[![Documentation](https://img.shields.io/badge/documentation-yes-brightgreen.svg)](https://github.com/the-dr-lazy/cascade/wiki)
+[![Twitter: the\_dr\_lazy\_](https://img.shields.io/twitter/follow/the\_dr\_lazy\_.svg?style=social)](https://twitter.com/the\_dr\_lazy\_)
+https://github.com/the-dr-lazy/cascade/blob/master/LICENSE
+> A project management service just to show that real-world Haskell is awesome.
+
+### 🏠 [Homepage](https://github.com/the-dr-lazy/cascade)
+
+### ✨ [Demo](www.demo.com)
+
+## Install
+
+```sh
+install command
+```
+
+## Usage
+
+```sh
+how to use it
+```
+
+## Run tests
+
+```sh
+how to test it
+```
+
+## Author
+
+👤 **Mohammad Hasani**
+
+* Website: https://the-dr-lazy.github.io/
+* Twitter: [@the\_dr\_lazy\_](https://twitter.com/the\_dr\_lazy\_)
+* Github: [@https:\/\/github.com\/the-dr-lazy](https://github.com/https:\/\/github.com\/the-dr-lazy)
+* LinkedIn: [@https:\/\/www.linkedin.com\/in\/the-dr-lazy](https://linkedin.com/in/https:\/\/www.linkedin.com\/in\/the-dr-lazy)
+
+## 🤝 Contributing
+
+Contributions, issues and feature requests are welcome!
+
+Feel free to check [issues page](https://github.com/the-dr-lazy/cascade/issues). You can also take a look at the [contributing guide](https://github.com/the-dr-lazy/cascade/graphs/contributors).
+
+## Show your support
+
+Give a ⭐️ if this project helped you!
+
+
+## 📝 License
+
+Copyright © 2021 [Mohammad Hasani](https://github.com/https:\/\/github.com\/the-dr-lazy).
```

Comment:
For now, this project is under my maintainership but I want to license it under the project name `Cascade`.